18-dualstack_ipv4_ipv6_seed_shoot.md

```diff
@@ -35,7 +35,47 @@ The easiest solution is to provide both protocol stacks - IPv4/IPv6 dual stack.
 
 ### General Considerations
 
-> TODO
+#### IPAM Setup and Process Flow
+
+This section describes the proposed configuration and process flow for the IP address management or more concrete how the CIDR ranges for nodes, pods and services are optained, configured and used during the cluster setup.
+
+> It has been seen that the different cloud provider does have quite strict limitations in a *bring-your-own-ipv6-ranges* scenario. Unless this has been changed we will mainly deal in this GEP with the situation, that the IPv6 ranges are given by the cloud provider.
+
+In the shoot specification we will use the definition of the `network` section to provide the cidrs for `nodes`, `pods` and `services`. As those fields already exist they will be enhanced to accept a tuple of values. One value for IPv4 and one for IPv6. For the IPv6 cidr the special key word `AUTOv6` will be valid to be used to indicate that the respective CIDR will be available at runtime only and being configured during the cluster deployment/reconciliation.
+
+Example:
+
+```yaml
+  networking:
+    type: calico # {calico,cilium}
+    pods: 100.128.0.0/11, AUTOv6
+    nodes: 10.255.0.0/16, AUTOv6
+    services: 100.72.0.0/13, AUTOv6
```

Comment:
As Services in K8S dual stack clusters are single stack by default and only use the first range specified, we should put IPv6 first here.
```suggestion
    pods: AUTOv6, 100.128.0.0/11, 
    nodes: AUTOv6, 10.255.0.0/16
    services: AUTOv6, 100.72.0.0/13
```